daemon/create.go

```diff
@@ -79,7 +85,7 @@ func (daemon *Daemon) containerCreate(params types.ContainerCreateConfig, manage
 }
 
 // Create creates a new container from the given configuration with a given name.
-func (daemon *Daemon) create(params types.ContainerCreateConfig, managed bool) (retC *container.Container, retErr error) {
+func (daemon *Daemon) create(params types.ContainerCreateConfig, managed bool, ignoreImagesArgEscaped bool) (retC *container.Container, retErr error) {
```

Comment:
Not a huge fan of the extra boolean argument (the `managed` was already a bit so-so). Perhaps it's time to introduce a `createOpts` struct (or functional arguments)

`daemon. CreateManagedContainer` and `daemon.ContainerCreate` could be moved to `_windows.go` and `_linux.go` files to pass different options / functional arguments

Reply:
I _knew_ you'd say that 😝. I really didn't want to tackle that, but I guess I know what I'll be doing after lunch today....

Reply:
Fixed in latest update.